nextjs/src/components/common/PasswordEyeButton.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
+import Image from 'next/image';
+
+const EyeIcon = styled.div`
+  width: 1.6rem;
+  height: 1.6rem;
+  position: relative;
+
+  button {
+    width: 100%;
+    height: 100%;
+    position: relative;
+  }
+`;
+
+interface PasswordEyeButtonProps {
+  inputType: string;
+  onToggle: () => void;
+}
+
+const PasswordEyeButton = ({ inputType, onToggle }: PasswordEyeButtonProps) => {
```

Comment:
inputType으로 분기하기보다 아래처럼 boolean값을(isHiden) 하나 둬서 어떻게 '보여줄지'만 책임지도록 하면 어떨까요?
어떤 경우에 가릴지는 사용처에서 결정하도록 하면 좋겠습니다.

```suggestion
interface PasswordEyeButtonProps {
  isHiden: boolean;
  onToggle: () => void;
}

const PasswordEyeButton = ({ isHiden, onToggle }: PasswordEyeButtonProps) => {
```